### ISSUE TITLE
ubuntu/deb support

### ISSUE THREAD

**Issue by turbo124** [toddstoffel/binlogrouter#1]
Appreciate the work on this!!!

Would it be relatively easy to port this for use on Ubuntu/Deb systems also?

**Comment by toddstoffel** [comment]
Yes, Ansible has an apt module as well. It should be fairly easy to replace the yum syntax in the playbook with apt syntax instead.


https://docs.ansible.com/ansible/latest/modules/apt_module.html <https://docs.ansible.com/ansible/latest/modules/apt_module.html>



Todd Stoffel
Senior Consultant - North America
MariaDB Corporation | t: +1 209 595 4339
